CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# [7.0.1]() (2020-03-30) (Jsonix)
```

Comment:
The Version is set later if we create a new one.
This should only be included in PR for new Versions see [publish a new version](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/DEVELOPMENT.md#then-the-ukis-team-will-publish-a-new-version)

Reply:
Got it! This line has been removed with commit [61a4d71](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/4/commits/61a4d7141436abd0eae67e658257bb239f9572f0)